.pre-commit-config.yaml

```diff
@@ -45,3 +45,10 @@ repos:
     hooks:
       - id: black
         name: black format code
+  - repo: local
+    hooks:
+    - id: pytest-testit
+      name: pytest-testit
+      language: script
+      types: [file, python]
+      entry: scripts/precommit/pytest-testit.bash
```

Comment:
hehe I feel a bit guilty for having added thisone

AFAIK these scripts act only on changed files, right? If that is the case, that approach can be useful to do some checks on requirements files that I cannot always enforce by code (e.g. do not install `respx` in base requreiments). I have added a standard test for it but I think it makes more sense to have it here (or perahps not?)

Reply:
might be a thing